docs/packages/pkg/nlohmann_fifo_map.rst

```diff
@@ -0,0 +1,21 @@
+.. spelling::
+
+    nlohmann
+    fifo_map
```

Comment:
Error: https://travis-ci.org/ruslo/hunter/builds/469294600#L2474

Should be:
```
 .. spelling::
 
    nlohmann
    fifo
    map
```

Reply:
done